cmd/pipeline/deploy.go

```diff
@@ -234,8 +236,21 @@ func (pc *Command) waitUntilRunning(ctx context.Context, name string, action *ty
 	signal.Notify(stop, os.Interrupt)
 	exit := make(chan error, 1)
 
-	go func() {
+	var wg sync.WaitGroup
```

Comment:
Can we use a `context.WithCancel()` instead. We need to cancel it when pressing `Ctrl+C` in the middle of the execution